scipy/stats/tests/test_multivariate.py

```diff
@@ -1635,6 +1638,187 @@ def test_haar(self):
+            [0, 1]
+        ],
+        # df
+        4,
+        # ans
+        [
+            0.013972450422333741737457302178882,
+            0.0010998721906793330026219646100571,
+            0.013972450422333741737457302178882,
+            0.00073682844024025606101402363634634,
+            0.0010998721906793330026219646100571,
+            0.0010998721906793330026219646100571,
+            0.0020732579600816823488240725481546,
+            0.00095660371505271429414668515889275,
+            0.00021831953784896498569831346792114,
+            0.00037725616140301147447000396084604
+        ]
+
+    ), (
+
+        # x
+        [
+            [0.9718, 0.1298, 0.8134],
+            [0.4922, 0.5522, 0.7185],
+            [0.3010, 0.1491, 0.5008],
+            [0.5971, 0.2585, 0.8940],
+            [0.5434, 0.5287, 0.9507],
+        ],
+        # mu
+        [-1, 1, 50],
+        # shape
+        [
+            [1.0000, 0.5000, 0.2500],
+            [0.5000, 1.0000, -0.1000],
+            [0.2500, -0.1000, 1.0000],
+        ],
+        # df
+        8,
+        # ans
+        [
+            0.00000000000000069609279697467772867405511133763,
+            0.00000000000000073700739052207366474839369535934,
+            0.00000000000000069522909962669171512174435447027,
+            0.00000000000000074212293557998314091880208889767,
+            0.00000000000000077039675154022118593323030449058,
+        ]
+    )]
+
+    @pytest.mark.parametrize("x, mu, df, shape, ans", MULTIVARIATE_T_TESTS)
+    def test_pdf_correctness(self, x, mu, df, shape, ans):
+        dist = multivariate_t(mu, df, shape, seed=0)
+        val = dist.pdf(x)
+        assert_array_almost_equal(val, ans)
+
+    @pytest.mark.parametrize("x, mu, df, shape, ans", MULTIVARIATE_T_TESTS)
+    def test_logpdf_correct(self, x, mu, df, shape, ans):
+        dist = multivariate_t(mu, df, shape, seed=0)
+        # We already test this elsewhere, but let's explicitly confirm the
+        # PDF is correct since this test requires checking the log of that
+        # value.
+        val1 = dist.pdf(x)
+        assert_array_almost_equal(val1, ans)
+        val2 = dist.logpdf(x)
+        assert_array_almost_equal(np.log(val1), val2)
+
+    # https://github.com/scipy/scipy/issues/10042#issuecomment-576795195
+    def test_mvt_with_df_one_is_cauchy(self):
+        x = [9, 7, 4, 1, -3, 9, 0, -3, -1, 3]
+        val = multivariate_t.pdf(x, df=1)
+        ans = cauchy.pdf(x)
+        assert_array_almost_equal(val, ans)
+
+    def test_mvt_with_high_df_is_approx_normal(self):
+        # `normaltest` returns the chi-squared statistic and the associated
+        # p-value. The null hypothesis is that `x` came from a normal
+        # distribution, so a low p-value represents rejecting the null, i.e.
+        # that it is unlikely that `x` came a normal distribution.
+        P_VAL_MIN = 0.1
+
+        dist = multivariate_t(0, 1, df=100000, seed=1)
+        samples = dist.rvs(size=100000)
```

Comment:
I'm just wondering what runtime of this one would be, maybe one of the 3 cases is sufficient?

Reply:
This is what I get using IPython's `%timeit`:

```
In [1]: from scipy.stats import multivariate_t
In [2]: %timeit dist = multivariate_t(0, 1, df=100000, seed=1); samples = dist.rvs(size=100000)
9.15 ms ± 73.2 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
```

We probably don't need all three though.

Reply:
Ok, if it's that quick we can safely leave as is, we do care about correct and coverage of tests but also are cognizant and make tradeoffs on speed for maintenance reasons as well-these can be left as is but maybe change the name of the test from `*_approx_normal` to `*_normal_or_approx`? 

Maintain test prefix string in place of `*`.

Reply:
Okay. Per [this reply](https://github.com/scipy/scipy/pull/12582#discussion_r492029144), I've opted to create two unit tests, `*_approx_normal` and `*_is_normal` for when `df` is large and `np.inf` respectively.

Reply:
marking resolved.